governance/bot-config.json

```
@@ -0,0 +1,18 @@
+{
+  "developerAbbreviation": "AE",
+  "protocolName": "Aave",
+  "protocolAbbreviation": "AAVE",
+  "contracts": {
+    "AaveGovernanceV2": {
+      "address": "0xEC568fffba86c094cf06b22134B23074DFE2252c",
+      "abiFile": "AaveGovernanceV2.json",
+      "events": [
+        "ProposalCreated",
```

This suggestion is for adding the remaining events that are part of the contract ABI.

I don't think that all of these events should have the same finding type and severity.  For example, if the ownership of the contract changes, that would warrant a more serious type and severity than when a new proposal is created.
```suggestion
        "ExecutorAuthorized"
        "ExecutorUnauthorized"
        "GovernanceStrategyChanged"
        "OwnershipTransferred"
        "VotingDelayChanged"
        "ProposalCreated",
```